.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build branches
+
+on:
+  push:
+    branches-ignore:
+      - 'master'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Scm
+        uses: actions/checkout@v2
+
+      - name: Set up JDK 12
```

Comment:
Why use java 12, and not 11 or 17 (LTS) ?

Reply:
copy/paste of release script to init this one.
I can do a lot of changes on the project but wanted to provide the minimal ones to have something testable ;-)

Reply:
@hervegareste updated to use jdk 17 in both scripts